bazel_tools/fat_cc_library.bzl

```diff
@@ -1,44 +1,60 @@
 # Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-load("@os_info//:os_info.bzl", "is_darwin")
+load("@os_info//:os_info.bzl", "is_darwin", "is_windows")
 
 def _fat_cc_library_impl(ctx):
     input_lib = ctx.attr.input_lib
     cc_info = input_lib[CcInfo]
-    pic_static_libs = []
+    static_libs = []
     # For now we assume that we have static PIC libs for all libs.
     # It should be possible to extend this but we do not have a need
     # for it so far and it would complicate things.
     for lib in cc_info.linking_context.libraries_to_link:
-        if not lib.pic_static_library:
-            fail("No PIC static lib for: " + lib)
```

Comment:
The `+ lib` caused a type error. `lib` is a Java object and can't be concatenated with a string. Also, it's string representation is not informative, so I've dropped it all together.

Reply:
Can we use something like lib.label? Some kind of hint as to what went wrong seems useful.

Reply:
Sadly there's no `.label` attribute. I've added the path to the dynamic library into the error message.

Reply:
:+1: That should be sufficient for debugging